.changelog/unreleased/features/provider/2035-min-stake-max-rank.md

```diff
@@ -0,0 +1,3 @@
+- Add minimum stake and maximum validator rank requirements to let consumer chains
+  determine requirements for validators that validate their chain. ([\#2035](https://github.com/cosmos/interchain-
+  security/pull/2035))
```

Comment:
Link probably needs to be in the same line. Here and in the file under `state-breaking/provider` as well.